packages/web-components/src/components/inline-loading/inline-loading.stories.ts

```diff
@@ -19,44 +20,127 @@ const states = {
   [`Failed (${INLINE_LOADING_STATE.ERROR})`]: INLINE_LOADING_STATE.ERROR,
 };
 
+const noop = () => {};
 const defaultArgs = {
   description: 'Loading data...',
   assistiveText: 'Loading',
+  iconDescription: 'Loading',
   status: INLINE_LOADING_STATE.ACTIVE,
+  successDelay: 2000,
 };
 
-const controls = {
-  description: {
-    control: 'text',
-    description: 'Specify the description for the inline loading text.',
-  },
-  assistiveText: {
-    control: 'text',
-    description:
-      'Specify a description that would be used to best describe the loading state.',
-  },
-  status: {
-    control: 'select',
-    description: 'Specify the loading status.',
-    options: states,
-  },
+const getControls = ({ disableControl }) => {
+  return {
+    description: {
+      control: disableControl || 'text',
+      description: 'Specify the description for the inline loading text.',
+    },
+    iconDescription: {
+      control: disableControl || 'text',
+      description:
+        'Specify a description that would be used to best describe the loading state.',
+    },
+    status: {
+      control: disableControl || 'select',
+      description: 'Specify the loading status.',
+      options: states,
+    },
+    onSuccess: {
```

Comment:
I think we can keep this out of the storybook controls, we generally don't add the custom events in the controls - they're documented in the storybook docs attribute, properties, and events table instead - generated by the web components analyzer

Reply:
I'm still seeing the `onSuccess` in the storybook controls, can we remove it?